plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqCodeTest/controller/CodeTestChatController.kt

```diff
@@ -637,7 +639,21 @@ class CodeTestChatController(
                         }
                         LOG.debug { "Original code content from reference span: $originalContent" }
                         withContext(EDT) {
-                            manager.addReferenceLogPanelEntry(reference = reference, null, null, originalContent?.split("\n"))
+                            // TODO flare: hook /test references with flare correctly, this is only a compile error fix which is not tested
```

Comment:
how should we track this

Reply:
we should add a line item in /test section in the doc